mininet/wifiMobility.py

```diff
@@ -180,7 +180,8 @@ def associate_wpa(self, sta, ap, wlan):
         :param wlan: wlan ID
         """
         passwd = self.verifyPasswd(sta, ap, wlan)
-        os.system('pkill -f \'wpa_supplicant -B -Dnl80211 -i %s\'' % sta.params['wlan'][wlan])
+        confname = "mn%d_%s_wifiDirect.conf" % (os.getpid(), sta.name)
```

Comment:
Ah now I got you, sorry for the misunderstanding. I will remove the config name, but I need another possibility to identify the correct process that should be killed. Because this:

```python
os.system('pkill -f \'wpa_supplicant -B -Dnl80211 -i %s\'' % sta.params['wlan'][wlan])
```
can kill the process of another instance of Mininet and another station with the same `sta.params['wlan'][wlan]` parameter, am I right?